Classes/Private/Manager/Database/OCTDBMessageCall.h

```diff
@@ -12,6 +12,7 @@
 @interface OCTDBMessageCall : RLMObject
 
 @property NSTimeInterval callDuration;
+@property OCTMessageCallType callType;
```

Comment:
It would be better to change `OCTMessageCallType` to `NSInteger` here. Realm is lower lever, we want to translate objcTox types to foundation data types here.